test/test_nestedtensor.py

```diff
@@ -141,6 +142,16 @@ def random_nt_from_similar(other, dims=None):
 
 
 class TestNestedTensor(TestCase):
+    def tearDown(self):
+        self._exit_stack.close()
+
+    def setUp(self):
+        self._exit_stack = contextlib.ExitStack()
+        self._exit_stack.enter_context(
+            unittest.mock.patch.object(torch._dynamo.config, "suppress_errors", False)
+        )
```

Comment:
Does it makes sense to factor these out somehow to prevent all the duplication?

Reply:
Good point, let me address this in a follow up.